needles/desktop_terminal2.json

```diff
@@ -10,6 +10,15 @@
       "width": 1024,
       "height": 728,
       "type": "match"
+    },
+    {
+      "": "Terminal window only match area",
+      "": "FIXME: This is a work around desktop shifting bug, https://phabricator.endlessm.com/T22227",
+      "xpos": 144,
+      "ypos": 82,
+      "width": 735,
+      "height": 481,
+      "type": "match"
```

Comment:
I’m not sure how this can be effectively working round T22227, given that there’s still also a 1024×728 match area around it (listed above in the JSON file).

Reply:
(Dropped)